### PR TITLE
pull Richie's drop down button for version selection

### PR DESCRIPTION
**Pull request by caesarxuchao**
Extracted from #16416

Tested on my local build and it works fine.

@nikhiljindal @janetkuo @bgrant0607

### CODE REVIEW

**Comment by caesarxuchao** [comment]
@nikhiljindal the problem we saw before noon was caused by my local configuration.

**Comment by caesarxuchao** [comment]
We need to test if sitemap is generated for v1.1 after this PR is merged.

**Comment by bgrant0607** [comment]
I already merged @janetkuo's addition of a link to the v1.1 docs.

File: _includes/docversionselector.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <div class="versionselector">
   <button>Version:</button>
   <ul class="dropdown-menu">
-    <li class="text-only">Latest:</li>
+    <li class="text-only">Others:</li>
```
Comment (bgrant0607) [inline _includes/docversionselector.html]: I think it would it make more sense to put the default at the top

File: _layouts/docwithnav.html [hunk 2]
```diff
@@ -16,6 +16,7 @@
 	<div class="row">
 		<!-- Navigation menu -->
 		<div class="col-md-3 hidden-sm hidden-xs" id="wrapper">
+            {% include docversionselector.html %}
```
Comment (bgrant0607) [inline _layouts/docwithnav.html]: Please match indentation

**Comment by caesarxuchao** [comment]
Updated. PTAL. Thanks.

**Comment by caesarxuchao** [comment]
Oh, then do we still need this PR? We can directly merge Richie's #16416 PR on Monday, that one LGTM.

**Comment by bgrant0607** [comment]
LGTM, thanks

**Comment by bgrant0607** [comment]
Someone should try the sitemap crawler to see if it finds v1.1.

**Comment by janetkuo** [comment]
I already tried the sitemap generator. It finds v1.1 now.

**Comment by caesarxuchao** [comment]
I think @janetkuo had tested with her #16606, sitemap would include v1.1

**Comment by bgrant0607** [comment]
Ok, I'm going to just close this, then. Thanks.

**Pull request closed without merging**